ir/base/cid.go

```diff
@@ -25,3 +26,7 @@ func (c Cid) WritePretty(w io.Writer) error {
 func (c Cid) MergeWith(ctx ir.MergeContext, x ir.Node) (ir.Node, error) {
 	panic("XXX")
 }
+
```

Comment:
why is this needed? Smart interface does not call for UnmarshalJSON method.

Reply:
It is not! Forgot to clean it from some tinkering I was doing. Thanks!